forms/aws/lambda/load_testing/locust_test_mail_list.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import random
+import uuid
+import json
+from json import JSONDecodeError
+from locust import HttpUser, task, between, events
+
+logging.basicConfig(level=logging.INFO)
+
+formIDs = ["81","83"]
+
+formSubmissions = {
+
+  "en":[
```

Comment:
need at least one test that triggers all 4 options at the same time

Reply:
How can all 4 options be triggered at the same time?

Reply:
All checkboxes are clicked